mathics/formatter/svg.py

```diff
@@ -312,22 +322,4 @@ def RoundBox_svg(self, offset=None):
         ry,
         style,
     )
-
-
-format2fn.update(
```

Comment:
@mmatera In this comment I start to DRY to code.  As with a number of other places where we DRY code, it does it by assuming some sort of convention.  Conventions and abbreviations abound in Physics for example, and if you don't understand the convention, then things are _more_ complex.

Therefore for experimental kinds of things I find it _useful_ to start out with the long form and then figure out how to shorten it.  In proofs, mathematicians and physicists are able to go immediately to the additional abbreviations and skip steps in proofs. I can't and find it helpful to start out with the long form and then compress from there. 

There still are two routines that need to get moved out of the Builtin code.  And I would like to start to work up a test for _just the conversion routines_. 

One other thing I love about this is that I now see the possibility of being about to write a utility function which I can use to write SVG. With this I can use this to experiment with  SVG's in a way that is in less heavy form than going through Mathics.

Reply:
If this is a temporary way to disentangle the code, I think it is OK. Also if at the end, we only expose to the core a function that convert an expression into an svg.